benchmarks/multimodal/sweep/orchestrator.py

```
@@ -0,0 +1,139 @@
+        print(f"\nGenerating plots -> {plots_dir}", flush=True)
+        generate_plots(
+            base_output_dir=file_output_dir,
+            output_dir=plots_dir,
+            benchmark_names=labels,
+        )
+    except ImportError:
+        print(
+            "WARNING: benchmarks.utils.plot not importable; skipping plots.",
+            flush=True,
+        )
+    except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)